src/_guides/language/analysis-options.md

```diff
@@ -214,6 +217,26 @@ In future, related lint rules may be coalesced into meta rules. See
 for more information.
 {% endcomment %}
 
+
+### Disabling individual rules
+
+If you include an analysis options file such as the one in `pedantic`, 
+you might want to disable some of the included rules.
+Disabling individual rules is just like enabling them,
+except that you add `: false` after the rule name.
+Here's an example of an analysis options file
+that uses all pedantic rules except `unawaited_futures`:
+
+{% prettify yaml %}
+include: package:pedantic/analysis_options.yaml
+
+linter:
+  rules:
+    - unawaited_futures: false
```

Comment:
The YAML syntax for key:value is a little different.  This should be:

```
linter:
  rules:
    unawaited_futures: false
```
---


Aside: you can think of the list format as shorthand for `rule:true`, which is to say

```
linter:
  rules:
    - unawaited_futures
```

is effectively equivalent to:

```
linter:
  rules:
    unawaited_futures: true
```

Reply:
Oh, cool. Obviously, I didn't test my code. Oops. I'll fix it.